components/camel-cxf/camel-cxf-soap/src/main/docs/cxf-component.adoc

```diff
@@ -576,10 +582,14 @@ Having simple java web service interface:
 ----
 package org.apache.camel.component.cxf.soap.server;
 
-@WebService(targetNamespace = "http://server.soap.cxf.component.camel.apache.org/", name = "EchoService")
-public interface EchoService {
+import jakarta.jws.WebService;
+
+@WebService(targetNamespace = "http://server.soap.cxf.component.camel.apache.org/", name = "TextService", serviceName = "TextService")
+public interface TextService {
 
-    String echo(String text);
+    String upperCase(String text);
+
+    String lowerCase(String text);
```

Comment:
```suggestion
    @WebMethod
    String upperCase(String text);

    @WebMethod
    String lowerCase(String text);
```

Plus the WebMethod import

Reply:
Why is it needed ? Just to showcase ? It works even without it.